packages/core/data/mm/world/overworld.yml

```diff
@@ -1925,29 +1960,26 @@
     "Stone Tower Pot Higher Scarecrow 9": "scarecrow_hookshot"
 "Stone Tower Front of Temple":
   region: STONE_TOWER
-  events:
-    MAGIC: "has_mask_goron || has_mask_zora || (scarecrow_hookshot && can_fight)"
-    BOMBS: "has_mask_goron || has_mask_zora || (scarecrow_hookshot && can_fight)"
-    ARROWS: "has_mask_goron || has_mask_zora || (scarecrow_hookshot && can_fight)"
-    RUPEES: "has_mask_goron || has_mask_zora || (scarecrow_hookshot && can_fight)"
```

Comment:
Did some research here and ended up creating new regions for the ledges as you suggested as it made the most sense. 

For the lower pots from STT:
- Zora can reach with a sidehop, no jump slash needed. Doesn't matter for logic either way.
- Jumping down and Hookshotting the scarecrow _does_ work.
- Human can simply backwalk off the top, jumpslash into the wall and recoil onto the ledge. Didn't include since, but could probably have as a trick in the future.
- There really aren't any bombs, arrows, or rupees in these pots. Added light arrow + keese

From the owl statue region, both ledges can be access with scarecrow + Hookshot following the route in the picture below.

![STScarewcrowLedges](https://github.com/OoTMM/OoTMM/assets/132797076/9b7da27a-cc61-46e5-9a9d-5f19a90d4e5e)

Both of those regions can also be accessed from in front of STT with the same items.